pkg/pod/pod.go

```diff
@@ -59,14 +60,20 @@ var (
 		Name:      "workspace",
 		MountPath: workspaceDir,
 	}, {
-		Name:      "home",
+		Name:      "tekton-home",
 		MountPath: homeDir,
+	}, {
+		// Mount the home Volume to both /tekton/home and (old,
+		// deprecated) /builder/home.
+		// After v0.10, we can remove this old path.
```

Comment:
would you mind adding an issue to track this, and adding the # here?

Reply:
Done! https://github.com/tektoncd/pipeline/issues/1633